### ISSUE TITLE
1.13 port

### ISSUE THREAD

**Issue by LuminaSapphira** [CerulanLumina/HealthHungerTweaks#18]
Porting to 1.13.
Status: **On Hold**
Waiting on:
- [x] Forge for 1.13
- [ ] AppleCore for 1.13

**Comment by LuminaSapphira** [comment]
1.14 is out, skipping 1.13
